src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -109,7 +109,15 @@ module sonic-portchannel {
                         }
                     }
                 }
-			} /* end of list PORTCHANNEL_LIST */
+
+                leaf tpid {
+                    type string {
+                        length 6;
```

Comment:
See, if we can move type to stypes.yang, and use that typedef here.

Reply:
Agreed. Will have this changed in next commit.